### ISSUE TITLE
geo_to_hca tools throws unboundLocalError

### ISSUE THREAD

**Issue by ESapenaVentura** [ebi-ait/hca-ebi-wrangler-central#448]
**Description of the issue**
When running the tool on the GEO_to_HCA repository, some accessions throw an error:
```
UnboundLocalError: local variable 'fastq_map' referenced before assignment
```

And the spreadsheet for that project does not generate

**How to reproduce this bug**
1. Clone the `geo_to_hca` repository or pull latest version
1. Run the following command: `python apps/geo_to_hca.py --accession GSE172176 --output_dir ~/geo_to_hca_test_area/`

**Comment by ami-day** [comment]
The reason why this error occurs is because there is no SRA study accession or open sequencing data for this project. On the GEO project page it says: *Submitter declares that the raw data were deposited in the European Genome-phenome Archive (EGA ID EGAS00001005258) due to patient privacy concerns*
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi

The tool requires both an SRA study ID and access to SRA experiment accessions. In this case they don't exist.

**Comment by ESapenaVentura** [comment]
Great catch @ami-day ! I will only add to this, that remember you can use the ingest UI to add projects by DOI, and it loads a lot of project info! (Although you need to check collaborators and fill a couple of fields out, but it saves a lot of time!)

**Comment by willrockout** [comment]
@ami-day @ESapenaVentura  I don't believe that is the issue except for in this case. I've had other GSE's that have had this error come up that have SRA Study IDs and experiment accession. Here are some examples GSE76381, GSE146123, and GSE176509.

**Comment by ami-day** [comment]
OK thanks Will I'll try running the script I have locally on those examples and see if I get this error, will get back
